### PR TITLE
Human Sprites 2: The Desimpsonifying

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Supercedes #6518.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!wip

File: code/modules/mob/living/carbon/human/appearance.dm [hunk 1]
```diff
@@ -22,9 +22,10 @@
 		return
 
 	src.gender = gender
+	dna.SetUIState(DNA_UI_GENDER, src.gender != MALE, 1)
 	reset_hair()
 	update_body()
-	update_dna()
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/appearance.dm]: hmm

File: code/modules/mob/living/carbon/human/species/station/human/human.dm [hunk 2]
```diff
@@ -25,7 +25,7 @@
 	name_language = null // Use the first-name last-name generator rather than a language scrambler
 	mob_size = 9
 	spawn_flags = CAN_JOIN
-	appearance_flags = HAS_HAIR_COLOR | HAS_SKIN_TONE | HAS_LIPS | HAS_UNDERWEAR | HAS_EYE_COLOR | HAS_SOCKS
+	appearance_flags = HAS_HAIR_COLOR | HAS_SKIN_TONE | HAS_LIPS | HAS_UNDERWEAR | HAS_EYE_COLOR | HAS_SOCKS | HAS_SKIN_COLOR
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/species/station/human/human.dm]: OH YES.

**Comment by NonQueueingMatt** [comment]
There will be presets for people to choose from when this is done. So no blue people unfortunately.

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/mob/living/carbon/human/appearance.dm (hunk 1 above)
Comment (Sindorman) [reply code/modules/mob/living/carbon/human/appearance.dm]: That's intentional

**Comment by Alberyk** [comment]
This breaks species that have body color selection, you can't pick it any skin color as a unathi, tajara, etc.

**Comment by NonQueueingMatt** [comment]
Stuff to fix:

- [ ] Xeno skin colors

- [ ] Make default skin color warm and 65

**Comment by ParadoxSpace** [comment]
Somewhy, this makes alien body colors off. You can't have red values below 37, or green/blue values below 2.

**Comment by NonQueueingMatt** [comment]
That is because I set the default skin color value for all humans to 37/3/2. I'll change it

**Comment by NonQueueingMatt** [comment]
The minimum will not be bumped down. You look like chalk with below 30.

**Pull request merged**